kv/tenant_test.go

```diff
@@ -0,0 +1,86 @@
+package kv_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/influxdata/influxdb"
+	"github.com/influxdata/influxdb/kv"
+	itesting "github.com/influxdata/influxdb/testing"
+	"go.uber.org/zap/zaptest"
+)
+
+func TestBoltTenantService(t *testing.T) {
+	t.Skip("some tests fail using `*kv.Service` as `influxdb.TenantService`, see the NOTEs in `testing/tenant.go`.")
+	itesting.TenantService(t, initBoltTenantService)
+}
+
+func TestBoltNewTenantService(t *testing.T) {
```

Comment:
this function should be in the tenant/tenant_test.go

Reply:
yes, you are right 😓